### DISCUSSION TITLE
When fine tuning, do I use the entire dataset or only the new images?

### DISCUSSION THREAD

**Discussion by sstainba** [ultralytics/yolov5#8974] [General]
So if I train a custom model and then acquire new images I want to fine tune with, do I provide the original dataset AND the new images when I do this or do I only use the new images?

**Comment by glenn-jocher** [comment]
@sstainba if the new data is on a different dataset then you simply use your new dataset.

**Reply by sstainba** [reply]
I assume that means I can also just add to the existing dataset (it's in Roboflow) and then train again on that same dataset but with the previous weights...

**Reply by glenn-jocher** [reply]
@sstainba yeah sure! You can start training from your pretrained weights, i.e. `python train.py --weights path/to/best.pt`
